erigon-lib/downloader/downloader.go

```diff
@@ -97,8 +98,15 @@ type Downloader struct {
 
 	stuckFileDetailedLogs bool
 
-	logPrefix string
-	startTime time.Time
+	logPrefix         string
+	startTime         time.Time
+	broadcast         func(name string, hash *prototypes.H160)
```

Comment:
can you rename this to onTorrentComplete

Reply:
Done